packages/deployments/contracts/contracts/core/connext/libraries/LibConnextStorage.sol

```diff
@@ -277,45 +277,51 @@ struct AppStorage {
   // 32
   mapping(bytes32 => mapping(address => uint8)) tokenIndexes;
   /**
-   * @notice Stores whether or not bribing, AMMs, have been paused.
+   * The address of an existing LPToken contract to use as a target
+   * this target must be the address which connext deployed on this chain.
    */
   // 33
+  address lpTokenTargetAddress;
+  /**
+   * @notice Stores whether or not bribing, AMMs, have been paused.
+   */
+  // 34
```

Comment:
what are these commented out numbers?

Reply:
dont worry, we will delete them in a docs PR that addresses fixes you all brought up during the audit, but we didnt have time to get through the fix / verification cycle. this pr will only address documentation

so that these numbers dont haunt you they were legacy, and maintained by habit. they are irrelevant and unhelpful at this point